Source/Extensions/Blazorise.Components/SelectList.razor.cs

```diff
@@ -80,6 +80,11 @@ public Task Focus( bool scrollToElement = true )
         /// </summary>
         [Parameter] public Func<TItem, TValue> ValueField { get; set; }
 
+        /// <summary>
+        /// Method used to determine if an item should be disabled.
+        /// </summary>
+        [Parameter] public Func<TItem, bool> DisabledField { get; set; }
```

Comment:
Since we're disabling a `SelectItem` I think a better name would be something along `ItemDisabled`. What do you think?

Reply:
I added the "Field" suffix to be consistent with `TextField`, `ValueField` which are `Func` parameters with similar purpose. Maybe `ItemDisabledField`?

Reply:
The `field` naming was meant for a field inside of a TItem.

Reply:
Ok, I'll rename to `ItemDisabled`.